embedchain/embedchain.py

```diff
@@ -521,17 +521,19 @@ def set_collection_name(self, name: str):
         # TODO: Check whether it is necessary to assign to the `self.collection` attribute,
         # since the main purpose is the creation.
 
-    def count(self) -> int:
-        """
-        Count the number of embeddings.
+    def db.count(self) -> int:
+        return self.db.count()
+  
+        # """
+        # Count the number of embeddings.
 
-        DEPRECATED IN FAVOR OF `db.count()`
+        # DEPRECATED IN FAVOR OF `db.count()`
 
-        :return: The number of embeddings.
-        :rtype: int
-        """
-        logging.warning("DEPRECATION WARNING: Please use `app.db.count()` instead of `app.count()`.")
-        return self.db.count()
+        # :return: The number of embeddings.
+        # :rtype: int
+        # """
+        # logging.warning("DEPRECATION WARNING: Please use `app.db.count()` instead of `app.count()`.")
+  
```

Comment:
Can you please remove the commented lines since we no longer will use it?

Reply:
@deshraj sure, done!